src/test/java/com/github/romankh3/image/comparison/ImageComparisonUtilUnitTest.java

```diff
@@ -22,13 +21,18 @@
 public class ImageComparisonUtilUnitTest {
 
     @Test
-    public void testFrameMethod() throws IOException, URISyntaxException {
+    public void testFrameMethod() throws IOException {
         BufferedImage image = readImageFromResources("result.png");
         Frame resultFrame = createGUI(image);
         assertEquals(image.getHeight(), resultFrame.getHeight());
         assertEquals(image.getWidth(), resultFrame.getWidth());
     }
 
+    @Test(expected = IOException.class)
+    public void testWrongPath() throws IOException {
```

Comment:
Please, add `//when-then` comment before line number 33.
It's BDD style, as you can see it's implemented for almost all the tests.